internal/cluster/installing.go

```diff
@@ -53,11 +53,11 @@ func (i *installingState) RefreshStatus(ctx context.Context, c *common.Cluster,
 func (i *installingState) getClusterInstallationState(ctx context.Context, c *common.Cluster, db *gorm.DB) (string, string, error) {
 	log := logutil.FromContext(ctx, i.log)
 
-	mappedMastersByRole := mapMasterHostsByStatus(c)
+	mappedMastersByRole := MapMasterHostsByStatus(c)
```

Comment:
Shouldn't the RefreshStatus above be removed? It should be handled by stateswitch

Reply:
yes